xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -278,9 +280,11 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the maximum number of TCP connections allowed to a single server.</summary>
```

Comment:
```suggestion
        <summary>Gets or sets the maximum number of simultaneous TCP connections allowed to a single server.</summary>
```
I guess we'll cross this bridge when we get to it, but might need to remove mention of TCP if we implement HTTP3.